docker-compose.yml

```
@@ -0,0 +1,13 @@
+version: '3'
+
+services:
+  redis:
+    image: redis:5.0.3-alpine
+    ports:
+      - "6379:6379"
```

```suggestion
      - "6379"
```
We don't need to expose redis to outside the swarm I think.